source/WebApp/scripts.ts

```diff
@@ -140,6 +142,26 @@ const latest = task('latest', () => jetpack.writeAsync(
     `${outputSharedRoot}/latest`, htmlOutputPath.replace(outputSharedRoot, '').replace(/^[\\/]/, '')
 ));
 
+const asm = task('asm', async () => {
+    const asmSourcePath = `${dirname}/components/internal/codemirror/mode-asm-instructions.txt`;
+    const asmOutputPath = `${dirname}/components/internal/codemirror/mode-asm-instructions.ts`;
+
+    // read list file as array of lines
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const listContents  = ((await jetpack.readAsync(asmSourcePath))!)
+        .split('\n');
+
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
+    const regexPattern: string = regexCombiner(listContents).toString();
+
+    const outputContents = [ "// This file is generated by 'asm' task defined in 'source/WebApp/scripts.ts'",
+        `export const instructionsRegex = new RegExp(${regexPattern});` ].join('\r\n');

```

Comment:
We are intentionally using CRLF in this repo (otherwise a linter rule violates based on .editorconfig rule). The template literal string, `` `${foo} ``, for some reason converts CRLF to LF, so I've used array with `.join('\r\n')` explicitly.